deploy/docker/fs/opt/appsmith/utils/bin/backup.js

```diff
@@ -207,7 +207,7 @@ function removeSensitiveEnvData(content) {
   // Remove encryption and Mongodb data from docker.env
   const output_lines = []
   content.split(/\r?\n/).forEach(line => {
-    if (!line.startsWith("APPSMITH_ENCRYPTION") && !line.startsWith("APPSMITH_MONGODB")) {
+    if (!line.startsWith("APPSMITH_ENCRYPTION") && !line.startsWith("APPSMITH_MONGODB") && !line.startsWith("APPSMITH_DB")) {
```

Comment:
For the other two, they're prefixes of _more than one_ env variable. But there's only one env variable that starts with `APPSMITH_DB`, so let's make it specific.

```suggestion
    if (!line.startsWith("APPSMITH_ENCRYPTION") && !line.startsWith("APPSMITH_MONGODB") && !line.startsWith("APPSMITH_DB_URL=")) {
```

Reply:
I thought in future we may want to move common envs from `APPSMITH_MONGODB*` to more generic `APPSMITH_DB*` hence kept only the prefix. But let's come back to this when required.